Ginger/GingerCoreNETUnitTest/LinuxTransformationTests/ActScriptTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.CoreNET.Repository;
+using Amdocs.Ginger.Repository;
+using GingerCore.Actions;
+using GingerCoreNETUnitTest.RunTestslib;
+using GingerTestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace GingerCoreNETUnitTest.LinuxTransformationTests
+{
+    [TestClass]
+    public class ActScriptTests
+    {
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            WorkSpace.Init(new WorkSpaceEventHandler());
+            WorkSpace.Instance.SolutionRepository = GingerSolutionRepository.CreateGingerSolutionRepository();
+        }
+        [TestMethod]
+        [Timeout(60000)]
+        public void VBSSum2ArgsTest()
+        {
+            //Arrange
+            ActScript actScript = new ActScript()
+            {
+                ScriptName = "VBSSum2Args.vbs",
+                ScriptCommand = ActScript.eScriptAct.Script
+            };
+            actScript.AddOrUpdateInputParamValueAndCalculatedValue("Var1", "56");
+            actScript.AddOrUpdateInputParamValueAndCalculatedValue("Var2", "77");
+            actScript.ScriptInterpreterType = ActScript.eScriptInterpreterType.VBS;
+            actScript.ScriptPath = TestResources.GetTestResourcesFolder(@"Files");
+            actScript.AddNewReturnParams = true;
+
+            //Act
+            actScript.Execute();
+
+            //Assert;
+            Assert.AreEqual(actScript.ReturnValues.Count, 3);
+            Assert.AreEqual(string.Join(',', actScript.ActReturnValues.Select(x => x.Param).ToList()), "Var1,Var2,sum");
+            Assert.AreEqual(string.Join(',', actScript.ActReturnValues.Select(x => x.Actual).ToList()), "56,77,133");
+        }
+
+        [TestMethod]
+        [Timeout(60000)]
+        public void BATFile1ArgTest()
+        {
+            //Arrange
```

Comment:
I see you added seperated Unit tests so try adding Retrun on begining of it if the OS version is Windows and same for VBS if it is NOT Windows

Reply:
you want only one test sperate to win/linux tests?